client/extensions/woocommerce/state/ui/shipping/zones/locations/selectors.js

```diff
@@ -113,30 +113,25 @@ const getStatesOwnedByOtherZone = createSelector(
 
```

Comment:
GitHub really messed up the diff in this file, maybe it's easier to just checkout the new code and see the new functions.